bears/python/PycodestyleBear.py

```diff
@@ -37,8 +37,10 @@ def create_arguments(
         """
         arguments = [r'--format=%(row)d %(col)d %(code)s %(text)s']
 
+        ignore_values = 'E121, E123, E126, E133, E226, E241, E242, E704, W503'
```

Comment:
You would do the same for your other PR modifying our coafile, you copy those defaults and append E303 (I think it was E303... anyway) for the `pycodestyle_ignore` setting.